engine/src/contract_methods/connector/mod.rs

```diff
@@ -199,7 +202,10 @@ pub fn finish_deposit<I: IO + Copy, E: Env, H: PromiseHandler>(
     #[cfg(not(feature = "ext-connector"))]
     let result = internal::finish_deposit(io, env, handler)?;
     #[cfg(feature = "ext-connector")]
-    let result = external::finish_deposit(io, env, handler)?;
+    let result = {
+        let (_, _, _) = (io, env, handler);
```

Comment:
It doesn't look good. I saw that pattern in other places in that PR as well. I understand that you want to be friendly with Clippy.

Reply:
Polished in https://github.com/aurora-is-near/aurora-engine/pull/1006/commits/80740e9dce52a44ce504297f631585a537c3f0ce